tests/e2e/scenarios/scalability/run-test.sh

```diff
@@ -86,7 +86,7 @@ echo "ADMIN_ACCESS=${ADMIN_ACCESS}"
 # cilium does not yet pass conformance tests (shared hostport test)
 #create_args="--networking cilium"
 create_args=()
-create_args+=("--networking=calico")
+create_args+=("--networking=${CNI_PLUGIN:-calico}")
```

Comment:
Use Ubuntu 20.04 as base, or make this also configurable:
```suggestion
create_args+=("--image=ssm:/aws/service/canonical/ubuntu/server/20.04/stable/current/arm64/hvm/ebs-gp2/ami-id")
create_args+=("--networking=${CNI_PLUGIN:-calico}")
```